pages/releases/4.3.html

```diff
@@ -1167,7 +1167,7 @@ <h4 class="c-title">
 									</a>
 								</h4>
 								<div class="c-blockquote">
-									<span class="highlight">frame rate</span> go brrr
```

Comment:
I feel this one is intentionally lower-case to be more "meme" like

Reply:
Fair enough! TBH, I wasn't really sure what "goes brr" was supposed to mean...